### DISCUSSION TITLE
Strange thing with Lookup time warnings

### DISCUSSION THREAD

**Discussion by galgool** [Chia-Network/chia-blockchain#5192] [Support]
Hey guys, maybe someone here can figure this out.
I’m having many harvester lookup time warnings that time is above 5 secs.
I have 330TiB. If I take it down to 250 TiB, I don’t get any warnings. But the more I add, the more frequently I get warnings.
I also added many usb splitters to spread out the hard drives, but nothing changed.
The strange thing is that every two\three plots in the warnings (from different hard drives, and even from different usb splitter) get EXACTLY the same lookup times. For example:

2021-05-15T13:35:31.992 harvester chia.harvester.harvester: WARNING  Looking up qualities on A:\final\plot-k32-2021-04-21-19-15-d838fa5819672b3d57208d3464b5f79cf5725054aa018281135f89550c45ee64.plot took: 5.674980878829956. This should be below 5 seconds to minimize risk of losing rewards.

2021-05-15T13:35:31.992 harvester chia.harvester.harvester: WARNING  Looking up qualities on N:\final\plot-k32-2021-01-01-05-11-e491f2526d61ef8dd64217bed20d6d128f2346872b8da2cd078a5103b7683741.plot took: 5.674980878829956. This should be below 5 seconds to minimize risk of losing rewards.

Any idea what’s the problem and how to fix?

**Comment by jjnst** [comment]
I have the exact same issue. I have swapped out all my hardware and used a brand new machine (HBA + SAS Expanders). Additionally, my disk utilization is extremely low. I haven't won a block with 800Tib + in over a week since I began getting this error. It appears to be software related as I've seen a delay larger than 300 seconds be returned from multiple drives across multiple independent HBA's / SAS expanders.

**Reply by galgool** [reply]
do you also get the duplicate lookup times?

In 16 days I won a block only once. 
it seems that all this started after upgrading from 1.1.1
You think we can go back to that version?

**Reply by jjnst** [reply]
I do get duplicate lookup times. I was constantly winning blocks until I went to version 1.1.4 and above, but I’m not sure if you can return to such an early version. People running 1.1.5 will likely automatically disconnect you if you forward negative balance transactions (Also I was told you are 30% less likely to win be an admin). I have a feeling there is a larger issue at play here. This is why the pooling update was pushed out. Many people who were previously farming just fine are encountering issues. The smaller farmers aren’t saying anything because their time to win is so long they simply think they are unlucky. I have been farming since test net, have not seen such a massive disruption.

**Reply by jjnst** [reply]
But to be fair the devs have already started working on some new beta versions that seem to have reduced my errors and as more and more people migrate to 1.1.5 you should have reduced signage errors.

**Reply by galgool** [reply]
What do you mean by larger issue at play?
And what’s this about the devs beta version that reduces your errors?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
